### ISSUE TITLE
VS2022 compatibilty

### ISSUE THREAD

**Issue by ilCosmico** [hamidshahid/shelvesetcomparer#19]
It would be really nice to have this extension available for Visual Studio 2022 as well.

**Comment by dprZoft** [comment]
Hi, I have created VS2022 compatible releases here: 
- ShelvesetComparer: https://github.com/dprZoft/shelvesetcomparer/releases/tag/v1.0.3.22
- Diff Finder: https://github.com/dprZoft/shelvesetcomparer/releases/tag/DiffFinder-v1.0.3.22

**Comment by ilCosmico** [comment]
Thanks a lot!
Unfortunately, I get an _Object reference not set_ error when trying to compare 2 shelvesets.
Could you check it, please?

![image](https://user-images.githubusercontent.com/8678145/155551455-40c843a5-a657-4dab-a741-96214a90a3bc.png)

**Comment by ilCosmico** [comment]
Hi @dprZoft , any news on this?

**Comment by dprZoft** [comment]
Hi, sorry I didnt have time to check it yet. It looks like sometimes the package is not yet initialized.
One possible workaround: use the Menu entries from the TEAMS menu in Visual Studio, these should ensure the package is loaded and thus the compare window will work.
I try to look into it next week.

**Comment by dprZoft** [comment]
Hi,

I have updated the VS2022 releases. I will try to merge it to the VS2019 releases also next week.
Can you please check the following (one of them):
* ShelvesetComparer: https://github.com/dprZoft/shelvesetcomparer/releases/tag/v1.0.4.22
* Diff Finder: https://github.com/dprZoft/shelvesetcomparer/releases/tag/DiffFinder-v1.0.4.22

**Comment by ilCosmico** [comment]
Hi @dprZoft they work like a charm! Thanks a lot!

**Comment by ilCosmico** [comment]
After updating Visual Studio 2022 to v17.1.3, I get the below error with _Compare Shelvesets_. 
_Diff Finder_ works fine instead.

![image](https://user-images.githubusercontent.com/8678145/162141092-41877b49-ea77-435f-a519-69317a018c1f.png)

**Comment by dprZoft** [comment]
Yes you are right, I never tested the installation of both extensions in parallel because I thought one or the other is enough :)
It looks like it can be fixed by generating new GUIDs for one of them, so I will create a new version of the DiffFinder which will work in parallel to the ShelvesetComparer.

You can find the updated version here: https://github.com/dprZoft/shelvesetcomparer/releases/tag/DiffFinder-v1.4.1.22

**Comment by ilCosmico** [comment]
The new version solves the issue, thanks!
